app/src/test/java/android/util/Log.java

```diff
@@ -0,0 +1,23 @@
+package android.util;
+
+public class Log {
```

Comment:
You could use [Timber](https://github.com/JakeWharton/timber) instead of building a new log library from the scratch

Reply:
That's a question always I ask to myself, do I use third libraries or not? Is It better use third library and keep focused where is really necessary? Maybe I can make this myself, I really don't know when I have to make the job easier and use a third library 🤔

Reply:
Yep, this is a good question. There is no right answer for that.

I always think this quote "_avoid to re-invent something that already exists_", and check if the library state is good or not, I mean checking:

- What is the date of the latest commit?

- How many developers have used this library?

For example, Timber library has:
- 9.5k stars 👍🏼 
- Last released made in less than one year 👍🏼 
- 47 contributors 👍🏼 

It is up to you to decide, make your decision 😉